org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageClientImpl.java

```diff
@@ -74,7 +74,11 @@ protected final LanguageServer getLanguageServer() {
 	@Override
 	public CompletableFuture<List<Object>> configuration(ConfigurationParams configurationParams) {
 		// override as needed
-		return CompletableFuture.completedFuture(Collections.emptyList());
+		List<Object> list = new ArrayList<>();
```

Comment:
we should correctly size the list when creating its object using `configurationParams.getItems().size()`, could you do that?

Reply:
done (but shouldn't really matter, these lists should always be tiny)

Reply:
Thanks. I realize that, it is mostly a matter of getting into a good habit :)